### DISCUSSION TITLE
découpe et la gravure en meme temps

### DISCUSSION THREAD

**Discussion by yann100775** [arkypita/LaserGRBL#1005] [Q&A]
j'adore LASERGRBL c'est un bon logiciel pour débuter
 mais 
j'ai un problème pour la découpe et la gravure en meme temps exemple comment faire pour découper l’extérieur en plusieurs passe 2.5w de "olaf" sans
 que je découpe "faire une gravure "  l’intérieur  merci

![images](https://user-images.githubusercontent.com/65447406/82117983-608bda00-9774-11ea-8a6d-ded735495296.jpg)

**Comment by Kalamgish** [comment]
Je pense que vous devez changez l'image par la main.
J'ai utilise inkscape pour ce but, si vous importez l'image puis changez-le en vecteurs vous pouvez le supprimer les lignes interieur.
